code/modules/clothing/under/accessories/armor.dm

```diff
@@ -0,0 +1,228 @@
+	icon_state = "armguards"
+	gender = PLURAL
+	body_parts_covered = ARMS
+	armor = list(melee = 40, bullet = 40, laser = 40, energy = 25, bomb = 30, bio = 0, rad = 0)
+	slot = ACCESSORY_SLOT_ARMOR_A
+
+/obj/item/clothing/accessory/armguards/blue
+	desc = "A pair of blue arm pads reinforced with armor plating. Attaches to a plate carrier."
+	icon_state = "armguards_blue"
+
+/obj/item/clothing/accessory/armguards/navy
+	desc = "A pair of navy blue arm pads reinforced with armor plating. Attaches to a plate carrier."
+	icon_state = "armguards_navy"
+
+/obj/item/clothing/accessory/armguards/green
+	desc = "A pair of green arm pads reinforced with armor plating. Attaches to a plate carrier."
+	icon_state = "armguards_green"
+
+/obj/item/clothing/accessory/armguards/tan
+	desc = "A pair of tan arm pads reinforced with armor plating. Attaches to a plate carrier."
+	icon_state = "armguards_tan"
+
+/obj/item/clothing/accessory/armguards/merc
+	name = "heavy arm guards"
+	desc = "A pair of red-trimmed black arm pads reinforced with heavy armor plating. Attaches to a plate carrier."
+	icon_state = "armguards_merc"
+	armor = list(melee = 60, bullet = 60, laser = 60, energy = 40, bomb = 40, bio = 0, rad = 0)
+
+//Leg guards
+/obj/item/clothing/accessory/legguards
+	name = "leg guards"
+	desc = "A pair of armored leg pads in black. Attaches to a plate carrier."
+	icon_override = 'icons/mob/modular_armor.dmi'
+	icon = 'icons/obj/clothing/modular_armor.dmi'
+//	accessory_icons = list(slot_tie_str = 'icons/mob/modular_armor.dmi', slot_wear_suit_str = 'icons/mob/modular_armor.dmi')
+	icon_state = "legguards"
+	gender = PLURAL
+	body_parts_covered = LEGS
+	armor = list(melee = 40, bullet = 40, laser = 40, energy = 25, bomb = 30, bio = 0, rad = 0)
+	slot = ACCESSORY_SLOT_ARMOR_L
+
+/obj/item/clothing/accessory/legguards/blue
+	desc = "A pair of armored leg pads in blue. Attaches to a plate carrier."
+	icon_state = "legguards_blue"
+
+/obj/item/clothing/accessory/legguards/navy
+	desc = "A pair of armored leg pads in navy blue. Attaches to a plate carrier."
+	icon_state = "legguards_navy"
+
+/obj/item/clothing/accessory/legguards/green
+	desc = "A pair of armored leg pads in green. Attaches to a plate carrier."
+	icon_state = "legguards_green"
+
+/obj/item/clothing/accessory/legguards/tan
+	desc = "A pair of armored leg pads in tan. Attaches to a plate carrier."
+	icon_state = "legguards_tan"
+
+/obj/item/clothing/accessory/legguards/merc
+	name = "heavy leg guards"
+	desc = "A pair of heavily armored leg pads in red-trimmed black. Attaches to a plate carrier."
+	icon_state = "legguards_merc"
+	armor = list(melee = 60, bullet = 60, laser = 60, energy = 40, bomb = 40, bio = 0, rad = 0)
+
+
+//Decorative attachments
+/obj/item/clothing/accessory/armor/tag
+	name = "\improper SCG Flag"
+	desc = "An emblem depicting the Sol Central Government's flag."
```

Comment:
Is this our flag or Bay's? Just asking so I know whether or not I should try and sprite a boi.

Reply:
I don't think those have sprites, but go ahead?

Reply:
I mean `icon_state = "solflag"` - at school right now so I can't check, just asking if it should go on my to-do list.

Reply:
Up to you, I don't care.